<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelFormat">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Start: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>End: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEditEnd">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Level: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxLevel">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Key: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditKey">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Value: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditValue">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEditSearch"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetKeyValues">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="toolButtonRemove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
       <item>
        <widget class="QLabel" name="lablePath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lableCount">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionDraw"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFormat"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Conigure</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
